src/pages/BaseResultsPage.js

```diff
@@ -3,98 +3,20 @@
 import React from "react";
 import { History } from "react-router";
 import reactMixin from "react-mixin";
-import _ from "underscore";
 
 import iss from "../iss";
 import BaseCategoriesPage from "./BaseCategoriesPage";
+import ResultsListPage from "./ResultsListPage";
+import ResultsMapPage from "./ResultsMapPage";
```

Comment:
It's going to be that much GC pressure vs everything else that creates GC pressure. We've never been concerned about this before. It's used approximately everywhere in React already.